Source/FakeItEasy.Specs/UnconfiguredFakeSpecs.cs

```diff
@@ -6,23 +6,28 @@
 
     public static class UnconfiguredFakeSpecs
     {
+        public interface IFoo
+        {
+            bool IsADummy { get; }
+        }
+
         [Scenario]
         public static void VirtualMethod(
             MakesVirtualCallInConstructor fake)
         {
-            "when faking a class with a virtual method"
+            "When faking a class with a virtual method"
                 .x(() => fake = A.Fake<MakesVirtualCallInConstructor>());
 
-            "it should return a default value when the method is called during the constructor"
+            "It should return a default value when the method is called during the constructor"

```

Comment:
No strong feeling about it but I think it can be left out. If the step just says "When I create a fake of the type" then expect it to do exactly that and nothing more. I don't think we need to call out what we are *not* doing.